### DISCUSSION TITLE
How to pass callables dynamically

### DISCUSSION THREAD

**Discussion by alysak6075** [spatie/fork#25] [Q&A] [answered]
I am having a hard time trying to pass callables dynamically to `run`. 
I keep receiving an error: `Argument #1 must be of type callable, array given,`

Any suggestions?

**Comment by sergio-paternoster** [comment]
Not sure I understand the question. Any code we can test?

**Reply by alysak6075** [reply]
actually figured it out, `run(...$array_of_callables())`, sorry to have bothered you.

**Comment by alysak6075** [comment, marked as answer]
`run(...$array_of_callables)` splash operator in the front fixes the issue.

**Comment by sergio-paternoster** [comment]
I think this could work as well, am I right?
```
call_user_func_array([Fork::new(), "run"], $array_of_callables);  
```

**Reply by alysak6075** [reply]
yes, I tried that too. the `...` just looks cleaner to me is all:)
